django_mongodb/compiler.py

```diff
@@ -64,7 +245,7 @@ def results_iter(
         return rows
 
     def has_results(self):
-        return bool(self.get_count(check_exists=True))
```

Comment:
`get_count`'s `check_exists` is no longer used. `has_results()` is now the same as the superclass so it can be removed here.